packages/node/src/index.ts

```diff
@@ -180,13 +180,16 @@ async function compile(
     return source;
   }
 
+  const conditions = isEdgeFunction ? ['worker', 'browser'] : undefined;
```

Comment:
Where did you find these conditions defined? I thought we should be using `edge-light` according to the standard https://runtime-keys.proposal.wintercg.org/

cc @Ethan-Arrowood who wrote the standard

Reply:
Ya we can add `edge-light`. Needs to be supported on the server-side still though, so we may want to wait for that and do in another PR.

Reply:
:+1: we should do that in another PR. We also need to update edge docs to include that that key should be added